src/options.rs

```diff
@@ -732,8 +735,26 @@ pub fn get_app_use_regex(matches: &ArgMatches, config: &Config) -> bool {
     false
 }
 
+pub fn get_use_terminal_name(matches: &ArgMatches, config: &Config) -> Option<String> {
+    match gethostname().into_string() {
```

Comment:
nit: IMO invert this - that is, check if the `match` or `config` have the corresponding flag first before running `gethostname`.